bescort.lic

```diff
@@ -610,8 +610,58 @@ class Bescort
     end
     case mode
     when /oasis/
-      echo 'Not implemented'
-      exit
+    loop do
```

Comment:
How confident are you that this will eventually terminate? I don't generally like unbounded `loop`s...

Reply:
I can't imagine it not terminating since it goes through every room until it finds the oasis and the oasis doesn't move too often, but I had it as a loop in the rare chance it moves while you are looking (happened during one of my test runs) but I ran it over 15 times and it found it and exited with the oasis in different locations. I could try and set it to only loop a few times. I just need to think of how to do that elegantly (still a new to ruby scripting).

Reply:
Nah, this seems fine with your explanation. Thank you!